courses/serializers.py

```diff
@@ -27,12 +29,23 @@ def get_programpage_url(self, program):
         except ProgramPage.DoesNotExist:
             return
 
+    def get_enrolled(self, program):
+        """
+        Returns true if the user
+        """
+        if 'request' in self.context:
```

Comment:
Going to remove this check based on @giocalitri's suggestion, since we never use this serializer without a user logged in